components/CompanyRegister.tsx

```diff
@@ -14,15 +15,15 @@ const CompanyRegister = () => {
   });
   const router = useRouter();
   const [companyName, setCompanyName] = useState('');
-  const [companyDescription, setCompanyDescription] = useState('');
```

Comment:
I think we will want to keep asking for company description so that possible in the future we will have a "About company" page...

Can ask @Hangout2698 see he wants it not...?

Reply:
checked in with @Hangout2698 on this one, companyDescription not required